src/or/dirvote_common.c

```diff
@@ -200,7 +200,7 @@ dirvote_get_voter_sig_by_alg(const networkstatus_voter_info_t *voter,
 /** Allocate and return a new authority_cert_t with the same contents as
  * <b>cert</b>. */
 authority_cert_t *
-authority_cert_dup(authority_cert_t *cert)
+dirvote_authority_cert_dup(authority_cert_t *cert)
```

Comment:
I think the original name for this function was correct, since it acts on authority_cert_t.  I think this whole commit should be removed or reverted.

Reply:
Yeah actually, that function is not longer needed outside `dirauth/dirvote.c` so it should go back there and thus untouched.

I can't revert the commit as it is with other things but I made an extra commit putting it back in `dirvote.c` with original name: bc393fc737